### ISSUE TITLE
values 

### ISSUE THREAD

**Issue by pippy360** [pippy360/workingFromDepthRepo#3]
100.0: precision @ 1 = 221.369                               
100.0: precision @ 1 = 238.022                               
100.0: precision @ 1 = 218.572                               
100.0: precision @ 1 = 218.572                               
100.0: precision @ 1 = 218.572                               
100.0: precision @ 1 = 215.268                               
100.0: precision @ 1 = 215.268                               
100.0: precision @ 1 = 215.268                               
100.0: precision @ 1 = 214.218                               
100.0: precision @ 1 = 214.218                               
100.0: precision @ 1 = 214.218                               
100.0: precision @ 1 = 214.218                               
100.0: precision @ 1 = 211.951                               
100.0: precision @ 1 = 213.432                               
100.0: precision @ 1 = 213.432                               
100.0: precision @ 1 = 213.953                               
100.0: precision @ 1 = 213.953                               
100.0: precision @ 1 = 224.033                               
100.0: precision @ 1 = 219.092                               
100.0: precision @ 1 = 225.090                               
100.0: precision @ 1 = 212.902                               
100.0: precision @ 1 = 212.902                               
100.0: precision @ 1 = 208.548                               
100.0: precision @ 1 = 208.548                               
100.0: precision @ 1 = 208.548                               
100.0: precision @ 1 = 208.548                               
100.0: precision @ 1 = 205.012                               
100.0: precision @ 1 = 217.009                               
100.0: precision @ 1 = 205.012                               
100.0: precision @ 1 = 205.575                               
100.0: precision @ 1 = 217.449                               
100.0: precision @ 1 = 205.575                               
100.0: precision @ 1 = 211.160                               
100.0: precision @ 1 = 211.160                               
100.0: precision @ 1 = 211.160                               
100.0: precision @ 1 = 211.160                               
100.0: precision @ 1 = 235.333                               
100.0: precision @ 1 = 223.419                               
100.0: precision @ 1 = 223.419                               
100.0: precision @ 1 = 236.240                               
100.0: precision @ 1 = 270.099                               
100.0: precision @ 1 = 236.240                               
100.0: precision @ 1 = 260.118                               
100.0: precision @ 1 = 247.584                               
100.0: precision @ 1 = 247.584                           
100.0: precision @ 1 = 247.584                
100.0: precision @ 1 = 253.402                
100.0: precision @ 1 = 281.637  

2017-11-07 09:39:29.886437: step 1110, loss = 299.60 (60.9 examples/sec; 0.131 sec/batch)                                                                                       [35/524]
output predict into refine                                                                                                                                                              
2017-11-07 09:39:32.390475: step 1120, loss = 245.91 (68.1 examples/sec; 0.117 sec/batch)                                                                                               
output predict into refine                                                                                                                                                              
2017-11-07 09:39:34.827351: step 1130, loss = 219.04 (32.0 examples/sec; 0.250 sec/batch)                                                                                               
output predict into refine                                                                                                                                                              
2017-11-07 09:39:37.210585: step 1140, loss = 194.23 (32.8 examples/sec; 0.244 sec/batch)                                                                                               
output predict into refine                                                                                                                                                              
2017-11-07 09:39:46.820958: step 1150, loss = 155.30 (33.6 examples/sec; 0.238 sec/batch)                                                                                               
output predict into refine                                                                                                                                                              
2017-11-07 09:39:48.364515: step 1160, loss = 154.33 (8.3 examples/sec; 0.961 sec/batch)                                                                                                
output predict into refine                                                                                                                                                              
2017-11-07 09:39:49.557686: step 1170, loss = 181.43 (51.9 examples/sec; 0.154 sec/batch)                                                                                               
output predict into refine                                                                                                                                                              
2017-11-07 09:39:50.771274: step 1180, loss = 214.90 (65.0 examples/sec; 0.123 sec/batch)                                                                                               
output predict into refine                                                                                                                                                              
2017-11-07 09:39:51.948080: step 1190, loss = 188.61 (68.0 examples/sec; 0.118 sec/batch)                                                                                               
output predict into refine                                                                                                                                                              
2017-11-07 09:40:05.404508: step 1200, loss = 142.04 (12.4 examples/sec; 0.647 sec/batch)                                                                                               
output predict into refine                                                                                                                                                              
2017-11-07 09:40:07.141621: step 1210, loss = 176.72 (9.8 examples/sec; 0.816 sec/batch)                                                                                                
output predict into refine                                                                                                                                                              
2017-11-07 09:40:08.318469: step 1220, loss = 196.14 (46.1 examples/sec; 0.173 sec/batch)                                                                                               
output predict into refine                                                                                                                                                              
2017-11-07 09:40:09.509189: step 1230, loss = 209.44 (68.0 examples/sec; 0.118 sec/batch)                                                                                               
output predict into refine                                                                                                                                                              
2017-11-07 09:40:25.511578: step 1240, loss = 198.49 (67.1 examples/sec; 0.119 sec/batch)                                                                                               
output predict into refine                                                                                                                                                              
2017-11-07 09:41:02.313888: step 1250, loss = 160.27 (2.0 examples/sec; 3.964 sec/batch)                                                                                                
output predict into refine                                                                                                                                                              
2017-11-07 09:58:53.609977: step 1260, loss = 145.85 (0.1 examples/sec; 107.511 sec/batch)                                                                                              
output predict into refine                                                                                                                                                              
2017-11-07 10:16:34.003983: step 1270, loss = 209.34 (0.1 examples/sec; 106.267 sec/batch)                                                                                              
output predict into refine                                                                                                                                                              
2017-11-07 10:34:22.622048: step 1280, loss = 153.02 (0.1 examples/sec; 106.806 sec/batch)                                                                                              
output predict into refine                                                                                                                                                              
2017-11-07 10:52:26.264987: step 1290, loss = 217.35 (0.1 examples/sec; 107.666 sec/batch)  
output predict into refine                    
2017-11-07 11:10:13.950744: step 1300, loss = 169.78 (0.1 examples/sec; 106.596 sec/batch)  
output predict into refine                    
2017-11-07 11:28:05.867479: step 1310, loss = 156.79 (0.1 examples/sec; 107.164 sec/batch)  
output predict into refine                    
2017-11-07 11:45:55.833164: step 1320, loss = 148.89 (0.1 examples/sec; 106.882 sec/batch)  
output predict into refine                    
2017-11-07 12:03:44.873092: step 1330, loss = 137.74 (0.1 examples/sec; 107.111 sec/batch)  
output predict into refine                    
2017-11-07 12:21:34.496910: step 1340, loss = 165.32 (0.1 examples/sec; 106.893 sec/batch)  
output predict into refine                    
2017-11-07 12:39:27.977725: step 1350, loss = 222.69 (0.1 examples/sec; 107.446 sec/batch)  
output predict into refine                    
2017-11-07 12:57:10.649312: step 1360, loss = 165.59 (0.1 examples/sec; 106.064 sec/batch)  
output predict into refine                    
2017-11-07 13:15:03.836793: step 1370, loss = 148.46 (0.1 examples/sec; 107.145 sec/batch)  
output predict into refine                    
2017-11-07 13:32:53.159001: step 1380, loss = 145.01 (0.1 examples/sec; 107.358 sec/batch)  
output predict into refine                    
2017-11-07 13:50:40.581402: step 1390, loss = 142.65 (0.1 examples/sec; 106.536 sec/batch)  
output predict into refine                    
2017-11-07 14:08:34.829485: step 1400, loss = 208.78 (0.1 examples/sec; 107.344 sec/batch)
